MinecraftClient/McTcpClient.cs

```diff
@@ -1664,5 +1664,20 @@ public void OnHeldItemChange(byte slot)
                 bot.OnHeldItemChange(slot);
             CurrentSlot = slot;
         }
+        
+        public void ChangeGamemode(int gamemode, Guid uuid)

```

Comment:
I think this method should be named `OnGamemodeChange` since it's an event callback.